content/english/python-pixel/Activities/activity-4.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Activity 4: Create Basic Filter"
+date: 2020-09-08T00:00:00Z
+weight: 4
+draft: false
+---
+
+<hr/>
+We have already talked about that the image on the screen is made of pixels. Now we can start to learn how to design your own filter on the image. Here are some examples of how to design a basic filter on your image.
+To prepare for the coming section, it is recommended that you review how to open and save an image.
+<br/><br/>
+You can jump to a specific section to review them:
+<br/>
+- <a href="../../../image-manipulation/open-image" target="blank">Open image</a>
```

Comment:
These should be also added in the pre-reqs field at the top of the file (see earsketch workshop for examples).